src/app/src/components/AnimatedArcsAndMap/AnimatedTotalSpendingBucket.tsx

```diff
@@ -36,7 +36,7 @@ export default function AnimatedTotalSpendingBucket({
             const totalAwardsAtTime: number = Object.values(
                 spendingAtTimeByState
             ).reduce((sum, stateResults) => {
-                sum += stateResults && stateResults.per_capita;
+                sum += (stateResults && stateResults?.total) || 0;
```

Comment:
```suggestion
                sum += stateResults?.total ?? 0;
```

Reply:
@jacobtylerwalls if we are seeing this on develop, is it worth spinning that out to it's own card?

Reply:
Yeah, that's wise. The spending codes "Z" and "1" are supposed to exclusively deal with BIL spending, so we need an investigation card to see if we botched something or if the API is returning bad results (or if a rogue agency submitted bad contract data, hah). Would you mind opening the card?

Reply:
Not at all. Will do so first thing.